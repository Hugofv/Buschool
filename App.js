import React, { Component } from 'react';
import { Router, Scene } from 'react-native-router-flux';

import Login from './components/view/login';

const App = () => {
  return (
    <Router>
      <Scene key="root">
        <Scene key="login"
          component={Login}
          hideNavBar
          initial
        />
      </Scene>
    </Router>
  );
}

export default App;