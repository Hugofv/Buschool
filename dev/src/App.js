import React, { Component } from 'react';
import { Router, Scene } from 'react-native-router-flux';
import { Provider } from "react-redux";
import configureStore from './store';
import { View } from "react-native";
import FlashMessage from "react-native-flash-message";

import Login from './components/view/login';
import Menu from './components/view/menu';
import Cadatro from './components/view/cadastro';
import Aluno from './components/view/aluno';
import Motorista from './components/view/motorista';
import Onibus from './components/view/onibus';

const store = configureStore();
const App = () => {
  return (
    <View style={{ flex: 1 }}>
      <Provider store={store}>
        <Router>
          <Scene key="root">
            <Scene key="login" component={Login} hideNavBar initial />
            <Scene key="menu" component={Menu} hideNavBar />
            <Scene key="cadastro" component={Cadatro} hideNavBar />
            <Scene key="aluno" component={Aluno} hideNavBar />
            <Scene key="motorista" component={Motorista} hideNavBar />
            <Scene key="onibus" component={Onibus} hideNavBar />
          </Scene>
        </Router>
      </Provider>
      <FlashMessage position="bottom" />
    </View>
  );
}

export default App;