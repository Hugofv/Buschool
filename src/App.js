import React, { Component } from 'react';
import { Router, Scene } from 'react-native-router-flux';
import { Provider } from "react-redux";
import configureStore from './store';

import Login from './components/view/login';
import Menu from './components/view/menu';
import Cadatro from './components/view/cadastro';
import Aluno from './components/view/aluno';
import Motorista from './components/view/motorista';
import Onibus from './components/view/onibus';

const store = configureStore();
const App = () => {
  return (
    <Provider store={store}>
      <Router>
        <Scene key="root">
          <Scene key="login" component={Login} hideNavBar initial/>
          <Scene key="menu" component={Menu} hideNavBar/>
          <Scene key="cadastro" component={Cadatro} hideNavBar/>
          <Scene key="aluno" component={Aluno} hideNavBar/>
          <Scene key="motorista" component={Motorista} hideNavBar/>
          <Scene key="onibus" component={Onibus} hideNavBar/>
        </Scene>
      </Router>
    </Provider>
  );
}

export default App;