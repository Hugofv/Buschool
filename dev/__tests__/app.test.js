import React from "react";
import { shallow } from "enzyme";
import configureStore from "redux-mock-store";
import App from "../src/App";

// imported as a connected component!
const middlewares = []; // you can mock any middlewares here if necessary
const mockStore = configureStore(middlewares);

const initialState = {};

describe("Testing App", () => {
  it("renders as expected", () => {
    const wrapper = shallow(<App />, {
      context: { store: mockStore(initialState) }
    });
    expect(wrapper.dive()).toMatchSnapshot();
  });
});
