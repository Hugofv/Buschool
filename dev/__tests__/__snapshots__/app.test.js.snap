// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing App renders as expected 1`] = `ShallowWrapper {}`;
